manifests/cluster-selector-crd.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Google LLC
+# Copyright 2023 Google LLC
```

Comment:
did the generator update this? it shoudn't update the year if the header already exists, but maybe the header doesn't already exist if the code that runs the generator deleted all the generated files first.

Reply:
Yes, same as other files. The generate recreates these manifests.